test/cppsuite/configs/background_compact_long.txt

```diff
@@ -0,0 +1,47 @@
+# Configuration for background_compact.
+duration_seconds=7200,
+cache_size_mb=10000,
+workload_manager=
+(
+    populate_config=
+    (
+        collection_count=100,
+        key_count_per_collection=1000000,
+        thread_count=10,
+        key_size=10,
+        value_size=1000
+    ),
+    insert_config=
+    (
+        thread_count=100,
```

Comment:
Depending on how much context switching you want, you could cut down this to like 40 or 50 or something. Realistically our hosts don't have that many cpus anyway. So the question is, is context switching relevant to your workload?

Reply:
Yeah that's probably more reasonable.

Reply:
Changed to 50 here https://github.com/wiredtiger/wiredtiger/pull/9691/commits/e9d1cf70dd6121186be8c96113da868100d3c775.